IDE/MonoDevelop/default.build

```diff
@@ -44,6 +49,9 @@
      <get src="https://www.nuget.org/api/v2/package/MonoGame.Framework.Content.Pipeline.Portable/3.2.99.1-Beta" dest="MonoDevelop.MonoGame/packages/MonoGame.Framework.Content.Pipeline.Portable.3.2.99.1-Beta.nupkg" />
      <get src="https://www.nuget.org/api/v2/package/Ouya.Console.Api/1.0.13.1" dest="MonoDevelop.MonoGame/packages/Ouya.Console.Api.1.0.13.1.nupkg"/>
      <if test="${os == 'MacOS'}">
+       <delete file="MonoDevelop.MonoGame_*.mpack" failonerror="false" />
```

Comment:
This doesn't seem to have worked... the files are still on the build agent.

I suspect that the `<delete>` task has to be used with a `<fileset>` to use wildcards.